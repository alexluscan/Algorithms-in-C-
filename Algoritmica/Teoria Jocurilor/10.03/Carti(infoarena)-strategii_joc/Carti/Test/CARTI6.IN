12
4 2
A 2 3 5 7 J
11 2
A 2 3 4 5 6 7 9 10 J K
11 2
A 2 3 5 6 7 8 9 10 J K
12 2
A 2 3 4 5 6 7 8 9 10 Q K
11 2
A 2 3 4 5 6 7 9 J Q K
11 2
A 3 4 5 6 7 8 9 J Q K
12 2
A 2 3 4 5 6 7 8 10 J Q K
12 2
A 2 3 4 5 6 8 9 10 J Q K
11 2
A 2 3 5 7 8 9 10 J Q K
11 2
A 3 4 5 7 8 9 10 J Q K
12 2
A 2 3 4 6 7 8 9 10 J Q K
12 2
A 2 4 5 6 7 8 9 10 J Q K
