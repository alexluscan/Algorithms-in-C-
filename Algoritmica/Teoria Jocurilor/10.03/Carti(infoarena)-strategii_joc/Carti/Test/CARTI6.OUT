Alice
Bob
Bob
Bob
Bob
Bob
Bob
Bob
Bob
Bob
Bob
Bob
